doc/api/crypto.md

```diff
@@ -703,18 +703,18 @@ If the `inputEncoding` is not provided, `key` is expected to be a [`Buffer`][],
 Example (uncompressing a key):
 
 ```js
-const { ECDH } = require('crypto');
```

Comment:
Couldn't this diff be simpler if you destructured out `ECDH` and `createECDH()` here?